Syllabus.md

```diff
@@ -0,0 +1,116 @@
+# GCB/CIS/MTR 535: Introduction to Bioinformatics
+
+## Course directors
+
+### Benjamin F. Voight, PhD
+Department of Systems Pharmacology and Translational Therapeutics
+Department of Genetics
+10-126 Smilow Center for Translational Research (SCTR)
+3400 Civic Center Blvd
+215-746-8083
+bvoight@upenn.edu
+
+### Casey S. Greene, PhD
+Department of Systems Pharmacology and Translational Therapeutics
+10-131 Smilow Center for Translational Research (SCTR)
+3400 Civic Center Blvd
+215-573-2991
+csgreene@upenn.edu
+
+## Teaching assistants
+
+* [Katie Siewert](mailto:ksiewert@mail.med.upenn.edu)
+* TBD
+
+### Guest proctors:
+
+* [Greg Grant](mailto:ggrant543@gmail.com)
+
+## Course web Site
+
```

Comment:
I think that we still wanted to use canvas for some elements as well. Was this for recording grades? If so, I can add that.